### PR TITLE
:sparkles: Creates App's website footer #135

### PR DESCRIPTION
**Pull request by dennis-mburu**
# Description

This PR creates a reusable footer which allows use of dynamic content depending on which page it will be used. No new dependencies have been added.

Fixes # 135

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Screenshot 
Screenshot from 2023-08-31 22-07-47
Screenshot from 2023-08-31 22-08-03
Screenshot from 2023-08-31 22-08-12

## How Has This Been Tested?
No test scripts added, just visual testing as illustrated on the screenshots above

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: libs/data/ui/slider-button/src/lib/mock-slider-button.data.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  color: 'white',
+  hoverBgColor: 'white',
+  hoverColor: 'black',
+  borderColor: '#3498db',
+  iconColor: '#3498db',
+  iconBackgroundColor: '#ffffff',
+  hoverBorderColor: '#2980b9',
+  hoverIconColor: 'red',
+  hoverIconBackgroundColor: 'black',
+};
```
Comment (Beulah-Matt) [inline libs/data/ui/slider-button/src/lib/mock-slider-button.data.ts:15]: This shouldn't be a module, just a ts file.

**Review by Beulah-Matt — CHANGES_REQUESTED** [review]
There are a lot of files here that came about by generating a module that are not needed. 
Look into that.

**Comment by dennis-mburu** [comment]
I have removed the unnecessary files and updated my code accordingly. Kindly review @Beulah-Matt

File: libs/elements/layout/footer/src/lib/components/footer/footer.component.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    <div class="ng-content">
+      <h1>
+        <!-- The ng-content is like a place holder and is where the footer's dynamic content will be rendered -->
+        <ng-content></ng-content>
+      </h1>
+
+      <div class="btn-container">
+        <elewa-website-slider-button
+          [buttonData]="buttonData"
+        ></elewa-website-slider-button>
```
Comment (LemmyMwaura) [inline libs/elements/layout/footer/src/lib/components/footer/footer.component.html:12]: The button should have been part of the dynamic content

**Review by LemmyMwaura — COMMENTED** [review]
Good job on the design. But I'm going to close this as it's already been solved

**Pull request closed without merging**